src/lib/utils.ts

```diff
@@ -1,11 +1,18 @@
-import { type ClassValue, clsx } from "clsx"
-import { twMerge } from "tailwind-merge"
-import { Locale } from '@/../i18n.config'
- 
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+import { Locale } from "@/../i18n.config";
+
 export function cn(...inputs: ClassValue[]) {
-  return twMerge(clsx(inputs))
+  return twMerge(clsx(inputs));
 }
 
-export const pathnameWithLocale = (locale: Locale, pathname: string) => (
-  `/${locale}${pathname.startsWith('/') ? '' : '/'}${pathname}`
-)
\ No newline at end of file
+export const pathnameWithLocale = (locale: Locale, pathname: string) =>
+  `/${locale}${pathname.startsWith("/") ? "" : "/"}${pathname}`;
+
+export const truncateAddress = (address: string) => {
+  return address
+    ? address.slice(0, 5) +
+        "..." +
+        address.slice(address.length - 5, address.length)
+    : "";
+};
```

Comment:
avoid `monolithic` shared `util` files
- please create a new `truncateAddress` lib in a STANDALONE file

rules
- less is more
- simple is best
- small is beautiful